README.md

```diff
@@ -190,36 +101,40 @@ If your font is not included in either of these locations you must include an hr
 
 Font support is a work in progress.
```

Comment:
Ditto. Instead state what works and what is not implemented. Don't make our readers guess;-)